packages/gatsby/lib/query-runner/page-query-runner.js

```diff
@@ -0,0 +1,120 @@
+/***
+ * Jobs of this module
+ * - Ensure on bootstrap that all invalid page queries are run and report
+ *   when this is done
+ * - Watch for when a page's query is invalidated and re-run it.
+ ***/
+
+const _ = require("lodash")
```

Comment:
👍 

Maybe it can be a good idea to add this "coding guidelines" into CONTRIBUTING.md (like https://github.com/lodash/lodash/blob/master/.github/CONTRIBUTING.md#coding-guidelines)